packages/block-library/src/site-logo/edit.js

```diff
@@ -154,14 +166,7 @@ const SiteLogo = ( {
 		);
 	}
 
-	let imageWidthWithinContainer;
-
-	if ( clientWidth && naturalWidth && naturalHeight ) {
-		const exceedMaxWidth = naturalWidth > clientWidth;
-		imageWidthWithinContainer = exceedMaxWidth ? clientWidth : naturalWidth;
-	}
-
-	if ( ! isResizable || ! imageWidthWithinContainer ) {
```

Comment:
I'm not sure I understood this condition. `imageWidthWithinContainer` returns either `clientWidth` or `naturalWidth`. Which will always be a number unless something went wrong with the image loading. So a check for `! naturalWidth || ! naturalHeight` should be enough here.